addon/components/o-s-s/checkbox.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="upf-checkbox {{this.modifierClasses}}" ...attributes {{on "click" this.noOp}}>
```

Comment:
`{{on "click" this.noOp}}`
Can't seem to find the noOp method anywhere, also what is it for ?